init-plugin.sh

```diff
@@ -26,8 +26,6 @@ prefix=$( perl -pe '$_ = lc; s/ /_/g' <<< "$name" )
 namespace=$( perl -pe 's/ //g' <<< "$name" )
 class=$( perl -pe 's/ /_/g' <<< "$name" )
 
-cd ..
```

Comment:
Why remove this and create the plugin inside the `foo-bar` directory? Is there a benefit or limitation that I'm not aware of? It's so much nicer to be able to activate the plugin immediately without having to perform any other commands or move the directory manually.